internal/tools/orchestrator/scheduler/instanceinfo/instance.go

```diff
@@ -164,7 +187,10 @@ func (r *InstanceReader) Limit(n int) *InstanceReader {
 }
 func (r *InstanceReader) Do() ([]InstanceInfo, error) {
 	instanceinfo := []InstanceInfo{}
-	expr := r.db.Where(strutil.Join(r.conditions, " AND ", true), r.values...).Order("started_at desc")
+	expr := r.db.Order("started_at desc")
+	for k, _ := range r.conditions {
+		expr = expr.Where(r.conditions[k], r.values[k])
```

Comment:
`ByHostIP` and `ByInstanceIP` query condition will be missing because it has multiple values